js/kucoin.js

```diff
@@ -786,6 +786,18 @@ module.exports = class kucoin extends Exchange {
         return (type === 'contract') || (type === 'future') || (type === 'futures'); // * (type === 'futures') deprecated, use (type === 'future')
     }
 
+    isMarginMethod (methodName, params) {
+        //
+        // Helper
+        // @methodName (string): The name of the method
+        // @params (dict): The parameters passed into {methodName}
+        // @return: true if the method used is meant for margin trading, false otherwise
+        //
+        const defaultType = this.safeString2 (this.options, methodName, 'defaultType', 'trade');
```

Comment:
`trade` is not a "unified" type/defaultType so I don't think we should be using it here because it might be confusing

Reply:
ok, i removed it.